CONTRIBUTING.md

```diff
@@ -110,6 +110,13 @@ We use [prettier][4] to format Markdown, JavaScript, and (most) HTML, in order
 to ensure a consistent style throughout our source. You can add prettier as a
 plugin in most development environments.
 
+Once it is installed, you can fix formatting by running the following in the
```

Comment:
@dol this might be something worth adding to your checklist (that formatting was run)

Reply:
Maybe also say something about IDE support/VS Code instructions?

Reply:
Added a link to the official page! good call.